packages/cloudbuster/src/digests.ts

```diff
@@ -15,18 +18,29 @@ export function createDigestsFromFindings(findings: Finding[]): Digest[] {
 }
 
 function createDigestForAccount(
-	awsAccountId: string,
+	accountId: string,
 	findings: GroupedFindings,
 ): Digest | undefined {
-	const teamFindings = findings[awsAccountId];
+	const teamFindings = findings[accountId];
 
 	if (!teamFindings || teamFindings.length == 0) {
 		return undefined;
 	}
 
+	const accountName = teamFindings[0]?.awsAccountName as string;
+
+	const actions = [
+		{
+			cta: `View all ${teamFindings.length} findings on Grafana`,
+			url: `https://metrics.gutools.co.uk/d/ddi3x35x70jy8d/fsbp-compliance?var-account_name=${accountName}`,
```

Comment:
```suggestion
			url: `https://metrics.gutools.co.uk/d/ddi3x35x70jy8d?var-account_name=${accountName}`,
```

The link still works without this string, and is slightly safer, as changing the name of the dashboard causes this string to change. It won't break anything, and isn't important enough to be a blocker, but might be slightly confusing for future travellers